src/ir/context.rs

```diff
@@ -929,6 +947,11 @@ impl<'ctx> BindgenContext<'ctx> {
             self.rust_ident_raw("std")
         }
     }
+
+    /// Whether a complex was detected anywhere
+    pub fn saw_complex(&self) -> bool {
+        return self.saw_complex.get();
```

Comment:
nit: no need for `return`, `self.saw_complex.get()` (without the semicolon) will do.